packages/space-opera/src/components/file_modal/file_modal.ts

```diff
@@ -60,14 +64,21 @@ export class FileModalElement extends LitElement {
 
   render() {
     return html`
-    <paper-dialog id="file-modal" modal ?opened=${this.show}>
-      <div>
-        <input type="file" class="input" id="file-input"
-          @change="${this.onFileChange}">
-        <mwc-button unelevated icon="cancel" @click=${
-        this.onCancel}>Cancel</mwc-button>
-      </div>
-    </paper-dialog>
+      <paper-dialog id="file-modal" modal ?opened=${this.show}>
+        <div class="FileModalContainer">
```

Comment:
Updating modal: https://screenshot.googleplex.com/6iRHM9jzkTBrEZU.png

Reply:
Bigger question: why do we need a dialogue here at all? The file picker already has a cancel button; can we just launch that directly from the "import GLB" button?

Reply:
this allows us to put some of the logic with respect to managing files into one place